lib/redirect.js

```diff
@@ -21,6 +21,13 @@ module.exports = function (redirects, port) {
       res.statusCode = redirect.code || 302;
       res.setHeader('Content-Type', 'text/plain');
       res.setHeader('Location', newUrl);
+
+      if(redirect.headers) {
+        for(var header in redirect.headers) {
```

Comment:
Make this the following:

`Object.keys(redirect.headers).forEach(function (header) {`